Track.h

```diff
@@ -109,12 +127,20 @@ class Track
 private:
   TrackState state_;
   HitVec hits_;
-  HitVec initHits_;
-  HitIdxVec hitIdxVec_;
   float chi2_;
+  unsigned int mcTrackID_;
+  unsigned int nHitsMatched_;
+  unsigned int seedID_;
+  unsigned int duplicateID_;
+  bool isDuplicate_;
+  HitIdxVec hitIdxVec_;
```

Comment:
@cerati @kmcdermo These should probably go into some kind of "track extra" structure, or in a specialized MCTrack class that refers to the Track class.  We should also discuss merging the data format reduction in the (currently mis-named) dsr -mpx-build branch, which has a stripped down Hit and Track class compatible with yours, but also an MCHit class that preserves the extra bookkeeping Kevin's validation wants.